uniprot-search/src/main/resources/taxonomy/search-fields.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "field": "name",
+    "sortField": "name"
```

Comment:
sortField should be name_sort

Reply:
Leo said to change it

Reply:
update, chatted with Leo, and an alternative, and possibly clearer solution is the following:
```
  {
    "field": "scientific",
    "sortField": "name_sort"
  },
```

Therefore, we will do this, and remove the `name` field from the config file.